crypto/bls/attestation.go

```diff
@@ -47,14 +47,20 @@ func NewG1Point(x, y *big.Int) *G1Point {
 	}
 }
 
+func NewZeroG1Point() *G1Point {
+	return NewG1Point(big.NewInt(0), big.NewInt(0))
+}
+
 // Add another G1 point to this one
-func (p *G1Point) Add(p2 *G1Point) {
+func (p *G1Point) Add(p2 *G1Point) *G1Point {
```

Comment:
What is the point of returning it if we are not creating a new object?

Reply:
This way you can chain Adds. Instead of having to do:
```
p1.Add(p2)
p1.Add(p3)
```
you can just do
```
p1.Add(p2).Add(p3)...
```

see for eg: https://github.com/Layr-Labs/eigensdk-go/blob/7172ee23384a8921450b45c8c8c3d123b2ad6422/services/bls_aggregation/blsagg_test.go#L103